server/models/PlantModel.js

```diff
@@ -51,7 +52,13 @@ const PlantSchema = new Schema({
     },
     problems: {
       type: [String],
+    },
+    care_level: {
```

Comment:
Don't go by me too much cos it's been a while, but I used to do camelCase with mongo too. Not sure if I was doing it right

Reply:
Good point. When I did it I wondered if camelCase would be better for consistency - I'll change it :)